src/anemoi/training/diagnostics/mlflow/logger.py

```diff
@@ -426,7 +432,71 @@ def _get_mlflow_run_params(
     def experiment(self) -> MLFlowLogger.experiment:
         if rank_zero_only.rank == 0:
             self.auth.authenticate()
-        return super().experiment
+
+        parent_obj = super()
```

Comment:
Just probably me not being familiar with this, could you explain what does `parent_obj = super()` do here?